frontend/src/components/DatePicker/DatePicker.tsx

```diff
@@ -18,12 +18,14 @@ export interface DatePickerProps extends DatePickerBaseProps, CalendarProps {
   onInputValueChange?: (value: string) => void
   /** Default value for uncontrolled input. */
   defaultInputValue?: string
+  /** Better visibility of values in the disabled component when set to true. */
+  isHighContrast?: boolean
 }
 
 export const DatePicker = forwardRef<DatePickerProps, 'input'>((props, ref) => {
   return (
     <DatePickerProvider {...props}>
```

Comment:
The isHighContrast comes from `<DatePickerProvider {...props}>` which is then referenced in `fcProps` and not `<DatePickerWrapper ref={ref} isHighContrast={props.isHighContrast}>`, which the isHighContrast is not being used.

Reply:
fixed in https://github.com/opengovsg/FormSG/pull/8251/commits/fef5d89dbce7f2cfc768db8517a5ca4bde276a93